dart/lib/src/sentry_client.dart

```diff
@@ -68,15 +72,26 @@ class SentryClient {
     return _options.transport.send(event);
   }
 
-  SentryEvent _prepareEvent(SentryEvent event) => event.copyWith(
-        serverName: event.serverName ?? _options.serverName,
-        dist: event.dist ?? _options.dist,
-        environment:
-            event.environment ?? _options.environment ?? defaultEnvironment,
-        release: event.release ?? _options.release,
-        sdk: event.sdk ?? _options.sdk,
-        platform: event.platform ?? sdkPlatform,
-      );
+  SentryEvent _prepareEvent(SentryEvent event) {
+    event = event.copyWith(
+      serverName: event.serverName ?? _options.serverName,
+      dist: event.dist ?? _options.dist,
+      environment:
+          event.environment ?? _options.environment ?? defaultEnvironment,
+      release: event.release ?? _options.release,
+      sdk: event.sdk ?? _options.sdk,
+      platform: event.platform ?? sdkPlatform,
+    );
+
+    if (event.throwable != null) {
```

Comment:
```suggestion
    if (event.throwable != null && event.exception == null) {
```

we should not overwrite an exception set by the client

Reply:
✅